src/odemis/driver/actuator.py

```diff
@@ -2359,7 +2355,7 @@ def _executeMoves(self, ordered_moves):
         Call the list of move functions in respect to their ordered execution
         :param ordered_moves: List of ordered moves functions and their arguments
         """
-        for i, (move_func, *args) in enumerate(ordered_moves):
+        for move_func, *args in ordered_moves:
```

Comment:
This `*args` is an invalid syntax on Python 2. So this prevent loading this module, which would be an issue. I'd suggest to pass the arguments for the function as (sub) tuple, so that you can write here just `move_func, args`.

In general, please check that the test cases pass with Python 2 too. That's not needed for new modules (because we are using Python 3 for any new system installation), but still needed on existing modules.